metadata-ingestion/src/datahub/utilities/sqlalchemy_type_converter.py

```diff
@@ -240,3 +246,25 @@ def get_schema_fields_for_sqlalchemy_column(
     )
 
     return schema_fields
+
+
+def get_native_data_type_for_sqlalchemy_type(
+    column_type: types.TypeEngine, inspector: Inspector
+) -> str:
+    if isinstance(column_type, types.NullType):
+        return column_type.__visit_name__
+
+    try:
+        return column_type.compile(dialect=inspector.dialect)
+    except Exception as e:
+        logger.warning(
```

Comment:
```suggestion
        logger.debug(
```

Reply:
@hsheth2 Just to mention it - it is not really important for me, but we probably should be consistent: A few lines above the same or let's say similar warning is a warning and not only a debug message...;-)

https://github.com/datahub-project/datahub/blob/89933fee1e141068fac60c3639eb8b2fa5b43871/metadata-ingestion/src/datahub/utilities/sqlalchemy_type_converter.py#L105-L107

https://github.com/datahub-project/datahub/blob/89933fee1e141068fac60c3639eb8b2fa5b43871/metadata-ingestion/src/datahub/utilities/sqlalchemy_type_converter.py#L210-L212